static/assets/js/filters.js

```diff
@@ -5,5 +5,12 @@ const showFilters = () => {
 
   document
     .querySelectorAll(".work__filter")
-    .forEach((button) => button.classList.toggle("work__filter--hidden"));
+    .forEach(
+      (button) => {
+        if(button.style.display == 'none') {
+          button.style.display = 'flex';
+        } else {
+          button.style.display = 'none';
```

Comment:
what is this for? This is already handled by applying the `work__filter--hidden` css class and toggled off and on with js

Reply:
When setting the `display` property of the `<li>` elements to `flex` it stopped the css toggle from working because the style engine picked up the `display` attribute from the base style class. The hidden class was being applied properly but because display was already set it was ignored. So by using the direct style method this solution guarantees the display attribute will have priority regardless of what is set in CSS.